airbyte-integrations/connectors/destination-bigquery/src/test-integration/kotlin/io/airbyte/integrations/destination/bigquery/BigQueryDestinationTest.kt

```diff
@@ -593,6 +601,8 @@ internal class BigQueryDestinationTest {
         protected var nonBillableConfig: JsonNode? = null
         protected var gcsStagingConfig: JsonNode? =
             null // default BigQuery config. Also used for setup/teardown
+        protected var gcsStagingConfigWithBadCopyPermission: JsonNode? = null
```

Comment:
nit: `protected lateinit var ... : JsonNode` to avoid the nullable type

(this is basically just promising the compiler that it'll never be null when you try to access it :P but junit provides that guarantee, since you're initializing this field in a BeforeAll function)

Reply:
Sure, this is a good idea, I have added lateinit and made the variable non-nullable

Reply:
did you mean to push a commit for this? github diff still shows a nullable var

Reply:
Spotbugs doesn't seem to like lateinit initialized in `@BeforeAll` because gen'd bytecode has a potential NPE.

Reply:
ugh. how do you feel about doing `@SuppressFBWarnings` for this?

also, it seems like spotbugs isn't even behaving consistently. E.g. this class (snowflake test-integration) has a lateinit var, immediately followed by a comment saying that spotbugs doesn't like lateinit... (and no `@Suppress` annotations anywhere) https://github.com/airbytehq/airbyte/blob/master/airbyte-integrations/connectors/destination-snowflake/src/test-integration/kotlin/io/airbyte/integrations/destination/snowflake/operation/SnowflakeStagingClientIntegrationTest.kt#L29-L33

(cc @stephane-airbyte this comment thread is a great example of why I think we should disable spotbugs :P We have three engineers trying to solve a spotbugs false positive, and now we're thinking about leaving a totally avoidable nullable variable because our tooling is actively getting in our way)

Reply:
I have added the lateinit and @SuppressFBWarnings

Reply:
The pipeline is failing with spotbugs error. Tried the variations listed below, still getting the error from spotbugs. Could you please share the suppression line to be added to resolve the error?

@SuppressFBWarnings
@Suppress("NP_NONNULL_RETURN_VIOLATION")
@SuppressFBWarnings("NP_NONNULL_RETURN_VIOLATION")

Reply:
I have created a new issue to track this problem as a separate item for all existing test code that could be changed to use lateinit

https://github.com/airbytehq/airbyte-internal-issues/issues/8931